src/reducers/entities/posts.js

```diff
@@ -511,7 +511,7 @@ export function postsInChannel(state = {}, action, prevPosts, nextPosts) {
         for (let i = order.length - 1; i >= 0; i--) {
             const postId = order[i];
 
-            if (nextPosts[postId].create_at <= mostRecentCreateAt) {
+            if (!nextPosts[postId] || nextPosts[postId].create_at <= mostRecentCreateAt) {
```

Comment:
Can you add a separate `if (...) continue` block for this with a separate comment explaining what's going on? The "This is an old post" comment doesn't apply here.

Reply:
@hmhealey done